proxysql/datadog_checks/proxysql/data/conf.yaml.example

```
@@ -0,0 +1,76 @@
+init_config:
+
+instances:
+    ## @param server - string - required
+    ## Proxysql host.
+    #
+  - server: "<SERVER_URL>"
+
+    ## @param port - integer - required
+    ## ProxySQL port for the admin interface.
+    #
+    port: <PROXYSQL_ADMIN_PORT>
+
+    ## @param user - string - required
+    ## Proxysql admin user. It must be able to connect remotely.
+    #
+    user: "<PROXYSQL_ADMIN_USER>"
+
+    ## @param pass - string - required
+    ## Proxysql admin password.
+    #
+    pass: "<PROXYSQL_ADMIN_PASSWORD>"
+
+    ## @param connect_timeout - integer - optional - default: 10
+    ## Timeout in seconds for connecting to ProxySQL.
+    #
+    # connect_timeout: <CONNECT_TIMEOUT>
+
+    ## @param read_timeout - integer - optional
+    ## Timeout in seconds for reading data from ProxySQL. Default "No Limit"
+    #
+    # read_timeout: <READ_TIMEOUT>
+
+    ## @param tags - list of key:value elements - optional
+    ## List of tags to attach to every metric, event and service check emitted by this integration.
+    ##
+    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param options - dictionary - optional
+    ## Activate additional metrics
+    #
+    # options:
+    #   ## @param extra_command_counters_metrics - boolean - optional - default: true
+    #   ## Send command_counters stats.
+    #   ##
+    #   ## Learn more about this metrics at: https://github.com/sysown/proxysql/wiki/STATS-(statistics)#stats_mysql_commands_counters
+    #   #
+    #   extra_command_counters_metrics: true
```

```suggestion
    #   extra_command_counters_metrics: true
   
```

I think adding a space after each config option will make it easier to read/configure.